### ISSUE TITLE
Updated build script to preserve code comments in SWC

### ISSUE THREAD

**Issue by BigPhilCombo** [tschneidereit/SwiftSuspenders#92]
By modifying the following node the in the build.xml file - code comments are added back to the generated SWC

```
  <!-- Build Documentation -->
  <target name="asdoc" depends="compile">
    <echo>[create-docs] Generating ASDOC documentation</echo>
    <tstamp>
      <format property="docgen.time" pattern="MM/dd/yyyy hh:mm aa" unit="hour"/>
    </tstamp>
      <asdoc output="${doc.loc}" lenient="true" failonerror="true" keep-xml="true" skip-xsl="true" fork="true">
          <compiler.source-path path-element="${main.src.loc}" />
          <doc-sources path-element="${main.src.loc}"/>         
      </asdoc>
      <zip destfile="${bin.loc}/${project.name.versioned}.swc" update="true">
          <zipfileset dir="${doc.loc}/tempdita" prefix="docs">
            <include name="*.*"/>
            <exclude name="ASDoc_Config.xml" />
            <exclude name="overviews.xml" />
          </zipfileset>
        </zip>
    <echo>[create-docs] ASDOC documentation generated successfully</echo>
  </target>
```

**Comment by tschneidereit** [comment]
Hey @pchertok, thanks for the patch. The reason your pull request had lots of other stuff in it is that this repository hasn't been updated in quite a while. SwiftSuspenders has officially [moved to the Robotlegs organization(https://github.com/robotlegs/swiftsuspenders), so this is mostly here for historical reasons.

I'll close this issue, please open one under https://github.com/robotlegs/swiftsuspenders.

**Comment by BigPhilCombo** [comment]
Strange,  I thought I opened it on the robotlegs branch.  I guess not.  My
bad.

On Thu, Jul 10, 2014 at 3:18 PM, Till Schneidereit <notifications@github.com

> wrote:
> 
> Closed #92 https://github.com/tschneidereit/SwiftSuspenders/issues/92.
> 
> —
> Reply to this email directly or view it on GitHub
> https://github.com/tschneidereit/SwiftSuspenders/issues/92#event-140338645
> .

## 

Phillip Chertok
